provision/provision.sh

```diff
@@ -12,6 +12,7 @@
 . "/srv/config/bash_aliases"
 
 # cleanup
+mkdir -p /vvv
```

Comment:
imo this should go in `/vagrant` and the move to `/vvv` should be a separate issue where we symlink it to `/vagrant`, that way we aren't spreading things across multiple folders